java/ingest/ingest-status-store/src/main/java/sleeper/ingest/status/store/task/DynamoDBIngestTaskStatusFormat.java

```diff
@@ -47,8 +47,8 @@ public class DynamoDBIngestTaskStatusFormat {
     public static final String FINISH_TIME = "FinishTime";
     public static final String MILLIS_SPENT_ON_JOBS = "MillisecondsOnJobs";
     public static final String NUMBER_OF_JOBS = "NumberOfJobs";
-    public static final String LINES_READ = "LinesRead";
-    public static final String LINES_WRITTEN = "LinesWritten";
+    public static final String RECORDS_READ = "RecordsRead";
+    public static final String RECORDS_WRITTEN = "RecordsWritten";
```

Comment:
Note this will break ingest task reporting for any existing deployed instance. It doesn't seem like a big problem though.